### PR TITLE
Fix error in current page example

### PR DESCRIPTION
**Pull request by wout**
I just noticed and error in the `current_page?` example where the links weren't wrapped in `<li>` tags

### CODE REVIEW

File: src/actions/guides/frontend/rendering_html.cr [hunk 1]
```diff
@@ -466,20 +466,19 @@ class Guides::Frontend::RenderingHtml < GuideAction
     ```crystal
     nav do
       ul do
-        link "Home",
-          to: Home::Index,
-          data_selected: current_page?(Home::Index)
+        li data_current: current_page?(Home::Index) do
+          link "Home", to: Home::Index,
```
Comment (jwoertink) [inline src/actions/guides/frontend/rendering_html.cr]:
```suggestion
          link "Home", to: Home::Index
```

File: src/actions/guides/frontend/rendering_html.cr [hunk 2]
```diff
@@ -466,20 +466,19 @@ class Guides::Frontend::RenderingHtml < GuideAction
...
+        end
 
-        link "Your dashboard",
-          to: Dashboard::Index,
-          data_selected: current_page?(Dashboard::Index)
+        li data_current: current_page?(Dashboard::Index) do
+          link "Your dashboard", to: Dashboard::Index,
```
Comment (jwoertink) [inline src/actions/guides/frontend/rendering_html.cr]:
```suggestion
          link "Your dashboard", to: Dashboard::Index
```

**Review by jwoertink — CHANGES_REQUESTED** [review]
Good catch! Just minor, but there's some trailing commas here.

File: src/actions/guides/frontend/rendering_html.cr (hunk 1 above)
Comment (wout) [reply src/actions/guides/frontend/rendering_html.cr]: Good catch!

**Review by jwoertink — APPROVED** [review]
:+1:

**Pull request merged**